utils/android/build_for_android.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+if [ ! $ANDROID_NDK_HOME ]; then
+    echo "env var ANDROID_NDK_HOME not set"
+    exit 1
+else
+    echo "use ndk: $ANDROID_NDK_HOME"
+fi
+
+WASMEDGE_ROOT_PATH=$(dirname $(dirname $(dirname $0)))
+
+cd ${WASMEDGE_ROOT_PATH}
+mkdir build
+cd build
+
+echo $(pwd)
+
+if ! cmake .. -DCMAKE_BUILD_TYPE=Release -DWASMEDGE_BUILD_AOT_RUNTIME=OFF -DCMAKE_SYSTEM_NAME=Android -DCMAKE_SYSTEM_VERSION=23 -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a -DCMAKE_ANDROID_NDK=$ANDROID_NDK_HOME -DCMAKE_ANDROID_STL_TYPE=c++_static -DWASMEDGE_BUILD_AOT_RUNTIME=OFF; then
```

Comment:
Please delete the `-DWASMEDGE_BUILD_AOT_RUNTIME=OFF` at the end of the line.

Besides, `CMAKE_SYSTEM_VERSION` should be set by user.

Reply:
The minimum Android API level requirement for WasmEdge is 23, only using `seekdir` func in bionic libc. Using a higher API level reduces system compatibility and provides no additional benefits.